src/utils/jss/ShadowRootPortal.tsx

```diff
@@ -3,7 +3,7 @@ import { untilDomLoaded } from '../dom'
 import { renderInShadowRootSettings } from '../../components/shared-settings/settings'
 
 const div = document.createElement('div')
-export const portalShadowRoot = div.attachShadow({ mode: 'closed' })
+export const portalShadowRoot = div.attachShadow({ mode: webpackEnv.shadowRootMode })
```

Comment:
Now you can set default of `renderInShadowRootSettings` to false in the test environment instead of add a new magic variable

Reply:
OK, `renderInShadowRootSettings` is great. I will remove the magic variable when switching the base to master.